sormas-api/src/main/resources/validations.properties

```diff
@@ -103,6 +103,7 @@ visitStatus = You have to specify the status for a visit.
 softApproximateAgeTooHigh = Please make sure that this approximate age value is correct; If applicable, change it to a value lower than 150.
 validPointOfEntry = You have to specify a valid point of entry
 exportNoNameSpecified = Please type in a name for your export configuration
+epiDataMultipleInfectionEnvironments = This epidemiological data already contains an exposure that has been marked as the probable infection environment. Setting this exposure as the probable infection environment will remove that mark from the other exposure. Do you want to set this exposure as the probable infection environment of this epidemiological data?
```

Comment:
Use case instead of epidemiological data, please. The checkbox actually has to be hidden for contacts anyway as I just realised (since contacts only have one exposure), but you've already implemented it that way.

Reply:
I have partially implemented that. The checkbox does not show in the ExposuresField for contacts, because I had the impression that it does not make sense there. It does show in the exposureForm for contacts though, as I thought there might be situations where this is helpful. Do you want me to completely hide it for contacts?
Another question: When converting a contact to a case, if that contact has an exposure, should this exposure automatically become the probable infection environment?

Reply:
Yes, it can be completely hidden from contacts (because since there can only be one exposure per contact, that's automatically the probable infection environment if it's transmitted to SurvNet)

For the second question, yes, I think that would be pretty helpful!